v2/internal/controllers/crd_batch_account_test.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package controllers_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+
+	batch "github.com/Azure/azure-service-operator/v2/api/batch/v1alpha1api20210101"
+	"github.com/Azure/azure-service-operator/v2/internal/testcommon"
+)
+
+func Test_Batch_Account_CRUD(t *testing.T) {
+	t.Parallel()
+
+	tc := globalTestContext.ForTest(t)
+
+	rg := tc.CreateTestResourceGroupAndWait()
+
+	// Custom namer because batch accounts have strict names
+	namer := tc.Namer.WithSeparator("")
```

Comment:
This seems to be a really common requirement, perhaps we should add a helper to the test context?

Reply:
I'll do this but in a separate PR.